components/registration/registration-controller.js

```diff
@@ -1422,31 +1422,31 @@ trackerCapture.controller('RegistrationController',
         return attributeMandatory || $scope.mandatoryFields[attributeId];
     }
 
-    $scope.labTestLookupNr = undefined;
-    $scope.labTestLookupNrGetterSetter = function(value) {
+    $scope.fNrOrEquivalent = undefined;
+    $scope.fNrOrEquivalentGetterSetter = function(value) {

```

Comment:
Do we ever need to "set" this? Can we just as well use the scope.getBestNumberForLabTestAndVaccine directly?